### PR TITLE
Feed language to form context instead of Custom mui dialog

### PR DESCRIPTION
**Pull request by Meklo**
Also adds missing updateElement API call necessary for description modal to work

### CODE REVIEW

File: src/components/dialogs/contingency-list/creation/contingency-list-creation-dialog.jsx
Comment (flomillot) [inline src/components/dialogs/contingency-list/creation/contingency-list-creation-dialog.jsx]: Language is not a form method and shouldn't be mixed with.
You need to change a bit commons-ui instead, and not this part.

```
        <CustomFormProvider
            {...formMethods}
            validationSchema={formSchema}
            removeOptional={removeOptional}
            **language={language}**
        >
```

**Review by flomillot — CHANGES_REQUESTED** [review]

File: src/components/dialogs/contingency-list/creation/contingency-list-creation-dialog.jsx
Comment (Meklo) [reply src/components/dialogs/contingency-list/creation/contingency-list-creation-dialog.jsx]: Alright it is done, here is the corresponding PR [#440](https://github.com/gridsuite/commons-ui/pull/440)

**Review by flomillot — APPROVED** [review]

**Comment by Meklo** [comment]
Waiting for next commons ui release before merge

**Pull request merged**
